setup.cfg

```diff
@@ -0,0 +1,55 @@
+[metadata]
+name = browser-history
+version = attr: browser_history.__version__
+description = A python module to extract browser history
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pesos/browser-history
+author = Samyak Sarnayak
+author_email = samyak201@gmail.com
+license = Apache License 2.0
+license_files = LICENSE
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+project_urls =
+    Documentation = https://github.com/pesos/browser-history/docs
+    Source = https://github.com/pesos/browser-history/
+    Tracker = https://github.com/pesos/browser-history/
+
+[options]
+include_package_data = true
+packages = find:
+python_requires = >=3.6
+zip_safe = false

```

Comment:
Is there a specific reason to set `zip_safe` to false here? According to [this](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#setting-the-zip-safe-flag), `browser-history` shouldn't be zip unsafe since we don't use C or perform any manipulations using `__file__` (except in the tests).

Reply:
Well are expecting it to run via the zips? If that's the case then it might not work. Also if we intend it run via the zips then its valid. What do you say? I can switch it the package demands it.

Reply:
I'm not entirely sure about this, I haven't come across this before. According to some sources, it generally doesn't matter with setuptools. So I don't see any harm in keeping it to `true`.